third-party/CMakeLists.txt

```diff
@@ -1,8 +1,13 @@
+option(USE_SYSTEM_PUGIXML "Use system pugixml library" OFF)
 
-add_library(pugixml STATIC
-		pugixml/src/pugiconfig.hpp
-		pugixml/src/pugixml.cpp
-		pugixml/src/pugixml.hpp)
-target_include_directories(pugixml PUBLIC pugixml/src)
-set_target_properties(pugixml PROPERTIES POSITION_INDEPENDENT_CODE ON)
-
+if (USE_SYSTEM_PUGIXML)
+	find_package(pugixml REQUIRED)
+	include_directories(${PUGIXML_INCLUDE_DIRS})
```

Comment:
Is this somehow necessary for you? The `pugixml` target exported by the `pugixml` package should already contain the include dir. Also, a `include_directories()` in this file does not affect the parent dirs at all, so it's probably a nop anyway.

Reply:
I guess you are right ;) I'm not very familiar with cmake. I have pushed an updated version without this line.